<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mac-Looker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>88</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan Network</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultPrompt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="networkScanLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Local Devices</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>14</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="macLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>44</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mac Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="macVendorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>73</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mac Vendor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipInput">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>10</y>
      <width>200</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="macAddressInput">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>40</y>
      <width>200</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="macVendorInput">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>70</y>
      <width>200</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>12</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionVendor_Search"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVendor_Search">
   <property name="text">
    <string>Vendor Search</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ipInput</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
